### DISCUSSION TITLE
piler demo has a nice balck theme.

### DISCUSSION THREAD

**Discussion by ckruijntjens** [jsuto/piler#60] [General]
Hi,

I upgraded to piler 1.4.5 and cant choose any themes besides default. I notice in the demo  there was a nice black theme. Is this not available in the community edition?

**Comment by jsuto** [comment]
Well, not yet. Do you want a toggle to switch between dark and light themes?

**Comment by ckruijntjens** [comment]
Hi,No the gui is different in the demo. Looks nicer. But if it cant be done its ok.Vriendelijke groet,Chris Kruijntjens.Van: jsuto ***@***.***> verzonden: donderdag 30 mei 2024 07:20 Aan: jsuto/piler Cc: ckruijntjens; AuthorOnderwerp: Re: [jsuto/piler] piler demo has a nice balck theme. (Discussion #60)
Well, not yet. Do you want a toggle to switch between dark and light themes?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/jsuto/piler/discussions/60#discussioncomment-9601347",
"url": "https://github.com/jsuto/piler/discussions/60#discussioncomment-9601347",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by jsuto** [comment]
Well, technically everything (almost) can be done. For the next release (1.4.6) the mission was to update the components (bootstrap and jquery). The demo site uses the commercial edition of piler that has bootstrap and vue.js 3. However, the layout can be updated for users to match the search page layout, however it would be a bigger job. I'll create a poll to ask the piler community to keep the current layout or switch to the one that can be seen on the demo site.

**Comment by jsuto** [comment]
Create a poll, https://github.com/jsuto/piler/discussions/61 please vote.

**Comment by ckruijntjens** [comment]
Hi,Perfect. We will se what everyone will vote. Thanks for the feedback.Vriendelijke groet,Chris Kruijntjens.Van: jsuto ***@***.***> verzonden: donderdag 30 mei 2024 09:09 Aan: jsuto/piler Cc: ckruijntjens; AuthorOnderwerp: Re: [jsuto/piler] piler demo has a nice balck theme. (Discussion #60)
Well, technically everything (almost) can be done. For the next release (1.4.6) the mission was to update the components (bootstrap and jquery). The demo site uses the commercial edition of piler that has bootstrap and vue.js 3. However, the layout can be updated for users to match the search page layout, however it would be a bigger job. I'll create a poll to ask the piler community to keep the current layout or switch to the one that can be seen on the demo site.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/jsuto/piler/discussions/60#discussioncomment-9602163",
"url": "https://github.com/jsuto/piler/discussions/60#discussioncomment-9602163",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
